sdk/package.json

```diff
@@ -14,7 +14,10 @@
   },
   "files": [
     "bundles",
-    "typings"
+    "typings",
+    "CHANGELOG.md",
```

Comment:
The license and readme files are included by default in npm packages, but the changelog is not. Listing all three files explicitly to make our intent clear.